src/Ng/App.config

```diff
@@ -10,7 +10,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-8.0.0.0" newVersion="8.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
```

Comment:
I don't think it's a very high priority, but if we're making changes in a project we might as well do so. The main reason I did this was because the compiler was telling me I needed to do this to fix a dependency--I think `NuGet.Services.Configuration` package required 9.x.